REQUIRE

```diff
@@ -1,3 +1,3 @@
-julia 0.4
-FITSIO 0.8.1
-Compat 0.8.6
+julia 0.7
+FITSIO 0.13.0+
```

Comment:
Why the `+`?  Isn't it for development version of packages?

Reply:
To be honest, I just looked at the package number I was using. We were working with a development version of that package for different reasons. I'll fix that.